managed/services/dbaas/kubernetes/client/client.go

```diff
@@ -974,7 +1011,7 @@ func (c *Client) CreateOperatorGroup(ctx context.Context, namespace, name string
 
 // CreateSubscriptionForCatalog creates an OLM subscription.
 func (c *Client) CreateSubscriptionForCatalog(ctx context.Context, namespace, name, catalogNamespace, catalog,
-	packageName, channel, startingCSV string, approval v1alpha1.Approval,
+	packageName, channel, startingCSV string, labels map[string]string, approval v1alpha1.Approval,
```

Comment:
Not something you need to change right now but just some food for thought for later. This is args list is getting quite long maybe we should think about creating a struct that logically groups some of them.

Reply:
Yes, I agree and in fact I did it and then I reverted my changes because probably this will be taken out of PMM and I didn't want to introduce more changes.

Reply:
There are several methods that are receiving a lot of parameters and I didn't want to add more package exported structures right now.